src/Components/NewBlog.js

```diff
@@ -1,41 +1,61 @@
-import axios from 'axios';
-import React, { useState } from 'react';
+import axios from "axios";
+import React, { useState } from "react";
 
-const CreatePost = () =>{
-  const [name, setName] = useState("");
-  const [title, setTitle] = useState("");
-  const [content, setContent] = useState("");
-  console.log(name);
+const CreatePost = () => {
+  const [post, setPost] = useState({
+    name: "",
+    title: "",
+    content: "",
+  });
 
-  const handleSubmit = (e) => {
+  const handleSubmit = async (e) => {
     e.preventDefault();
-    axios.post("https://blogpostapi1.herokuapp.com/", {
-      name,
-      title,
-      content,
-      }
-    )
+    try{await axios.post("https://blogpostapi1.herokuapp.com/", {
+      name:post.name,
+      title:post.title,
+      content:post.content
```

Comment:
Since they `post` and your `POST data` share the same structure, you can just pass `post` instead of spelling each one out

Reply:
```jsx
const post  = {name:"", title:"", content:""};

const payload = {name:"", title:"", content:""}

post === payload
```

You can just do `axios.post(url, post)` instead